src/aioftp/client.py

```diff
@@ -1155,13 +1338,18 @@ async def get_passive_connection(
                 is_last = i == len(commands)
                 if is_last or not e.received_codes[-1].matches("50x"):
                     raise
-        if ip in ("0.0.0.0", None):
+        if ip in ("0.0.0.0", None):  # type: ignore
```

Comment:
What do we do with this one? Remove type: ignore and add a check with raise throw?

Reply:
Nah, it is a logic issue with checkers. This can be ignored.